### PR TITLE
minor bug fixes

### PR DESCRIPTION
**Pull request by jxmorris12**
- fix flair version bug and fix version; 
- fix isort version bug and fix verison;
- fix alzantot abstract constraint class bug

### CODE REVIEW

File: textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py [hunk 1]
```diff
@@ -75,6 +78,9 @@ def get_probs(current_text, transformed_texts):
             probs_of_swaps_at_index = list(
                 zip(item_indices, get_probs(current_text, this_transformed_texts))
             )
+            # if len(probs_of_swaps_at_index) == 0:
+            #     probs.extend(0)
+            #     continue
```
Comment (heytitle) [inline textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py:83]: if we don't need this block here, maybe we can remove it?

File: textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py [hunk 2]
```diff
@@ -75,6 +78,9 @@ def get_probs(current_text, transformed_texts):
             probs_of_swaps_at_index = list(
                 zip(item_indices, get_probs(current_text, this_transformed_texts))
             )
+            # if len(probs_of_swaps_at_index) == 0:
+            #     probs.extend(0)
```
Comment (jinyongyoo) [inline textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py:82]: Should this be commented?

File: textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py (hunk 1 above)
Comment (jxmorris12) [reply textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py:83]: thanks! will delete

File: textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py (hunk 2 above)
Comment (jxmorris12) [reply textattack/constraints/grammaticality/language_models/google_language_model/google_language_model.py:82]: thanks! will delete

**Pull request merged**